e2e/support/test_support.go

```diff
@@ -1151,12 +1234,12 @@ func Platform(ns string) func() *v1.IntegrationPlatform {
 					continue
 				}
 				if pl != nil {
-					panic("multiple primary integration platforms found in namespace " + ns)
+					failTest(fmt.Errorf("multiple primary integration platforms found in namespace " + ns))
 				}
 				pl = &p
 			}
 			if pl == nil {
-				panic(fmt.Sprintf("multiple integration platforms found in namespace %q but no one is primary", ns))
+				failTest(fmt.Errorf(fmt.Sprintf("multiple integration platforms found in namespace %q but no one is primary", ns)))
```

Comment:
Same here. You don't need `fmt.Springf`.
```
fmt.Errorf("multiple integration platforms found in namespace %q but no one is primary", ns)
```

Reply:
Done